components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/policy/PolicyViolationException.java

```diff
@@ -28,6 +28,10 @@ public PolicyViolationException(String message) {
         super(message);
     }
 
+    public PolicyViolationException(String errorCode, String message) {
+        super(errorCode, message);
```

Comment:
```suggestion
        
        super(errorCode, message);
```

Reply:
And add a method description for newly adding methods

Reply:
These lines of code define a new constructor for the PolicyViolationException class, enabling the creation of exceptions with error message and error code as a use case of constructor overloading.